test/unit/lib/fetcher.spec.ts

```diff
@@ -494,4 +494,55 @@ describe('Fetcher', function () {
     const result = ['a', 'b', 'c', 'd', 'e', 'f']
     expect(localeCodes).to.eql(result)
   })
+
+  it('fetches all tags in the enviroment', async function () {
+    const intents = await buildIntents(function up (migration) {
+      migration.createTag('person', {
+        name: 'bar',
+        description: 'A content type for a person'
+      })
+    }, noOp, {})
+
+    const request = sinon.stub()
+    request
+      .withArgs({
+        method: 'GET',
+        url: `/tags?limit=${Fetcher.perRequestLimit}&order=sys.createdAt&skip=0`
```

Comment:
Does the tags API actually support `order` right now without erroring?

Reply:
My manual tests suggested that it does, although the parameter is being ignored.